tests/unit/lms/views/api/exceptions_test.py

```diff
@@ -62,6 +62,7 @@ def test_it(self, context, pyramid_request, views, report_exception, sentry_sdk)
         assert json_data == {
             "message": context.message,
             "details": {
+                "cause": "None",
```

Comment:
Could we add a `__cause__` to the test exception so that we test the exception view's cause rendering?

```diff
diff --git a/tests/unit/lms/views/api/exceptions_test.py b/tests/unit/lms/views/api/exceptions_test.py
index 64def955..3063b96f 100644
--- a/tests/unit/lms/views/api/exceptions_test.py
+++ b/tests/unit/lms/views/api/exceptions_test.py
@@ -62,7 +62,7 @@ class TestExternalRequestError:
         assert json_data == {
             "message": context.message,
             "details": {
-                "cause": "None",
+                "cause": "ValueError('foo')",
                 "request": {
                     "method": context.method,
                     "url": "https://example.com/",  # The URL without query string.
@@ -85,7 +85,7 @@ class TestExternalRequestError:
 
     @pytest.fixture
     def context(self):
-        return ExternalRequestError(
+        context = ExternalRequestError(
             message="test_message",
             request=requests.Request(
                 "GET", "https://example.com?foo=bar", data="request_body"
@@ -95,6 +95,8 @@ class TestExternalRequestError:
             ),
             validation_errors=sentinel.validation_errors,
         )
+        context.__cause__ = ValueError("foo")
+        return context
 
 
 class TestNotFound:
```